e2e/components/sidenav/sidenav.ts

```diff
@@ -108,7 +108,7 @@ export class Sidenav extends Component {
   private getLinkLabel(name: string): ElementFinder {
     switch (name) {
       case 'Personal Files': return this.personalFiles;
-      case 'File Libraries': return this.fileLibraries;
```

Comment:
this should probably remain unchanged. This refers to the sidenav link, which is called File Libraries, the parent of My Libraries and Favorite Libraries